src/transformers/modeling_t5.py

```diff
@@ -629,7 +629,6 @@ def forward(
                 all_attentions = all_attentions + (layer_outputs[1],)  # We keep only self-attention weights for now
 
         hidden_states = self.final_layer_norm(hidden_states)
-        layer_output = self.dropout(hidden_states)
```

Comment:
I'm not sufficiently familiar with PyTorch to be sure this has no side effects.

I believe the intent is to generate an output layer by applying dropout to the hidden states, and since the output layer isn't actually used, it's safe to remove this operation.

Reply:
This dropout should actually be used (cf. https://github.com/tensorflow/mesh/blob/master/mesh_tensorflow/transformer/transformer.py#L340)
It has a very small effect though but I'll add it for correctness.